lib/lens/event_formatter.rb

```diff
@@ -1,9 +1,10 @@
 module Lens
   class EventFormatter
-    def initialize(event, records, gc_time = 0.0)
+    def initialize(event, records, gc_time = 0.0, allocations_data = Lens::AllocationsData.new)
```

Comment:
Unused method argument - `gc_time`. If it's necessary, use `_` or `_gc_time` as an argument name to indicate that it won't be used.<br>Line is too long. [95/80]